{{cookiecutter.project_slug}}/Makefile

```diff
@@ -143,7 +147,7 @@ jupyter: upload-code upload-notebooks ### Run a job with Jupyter Notebook and op
 		--volume $(RESULTS_PATH_STORAGE):$(RESULTS_PATH_ENV):rw \
 		--env PLATFORMAPI_SERVICE_HOST="." \
 		$(CUSTOM_ENV_NAME) \
-		'jupyter notebook --no-browser --ip=0.0.0.0 --allow-root --NotebookApp.token= --notebook-dir=$(NOTEBOOKS_PATH_ENV)'
+		'jupyter notebook --no-browser --ip=0.0.0.0 --allow-root --NotebookApp.token= --notebook-dir=$(PROJECT_PATH_ENV)'
```

Comment:
we should start notebook for the project root directory so that the imports like `from modules import *` work

Reply:
Our current setup is too flexible for such a change. You may theoretically put your notebooks in some different location that '$(PROJECT_PATH_ENV)/notebooks'; running Jupyter like this will make them unavailable.

We should either find a way to make modules accessible without this change, or make the template structure less flexible.

Reply:
That's why this PR changes notebooks' working directory to the project root

Reply:
As @asvetlov has pointed out, we should not create a Python package for the users (using `setup.py`, see PR https://github.com/neuromation/cookiecutter-neuro-project/pull/107) because Python packaging is for *distributing* Python code, but in order to run it locally we can simply use `import modules` while running from the project root directory. As far as I understood, `pip install -e .` makes unnecessary changes to the environment (also, our `setup.py` was incorrect). So we should better enforce our notebooks to have working directory = project root. See also comment https://github.com/neuromation/cookiecutter-neuro-project/pull/97/files#r329563695

Reply:
> We should either find a way to make modules accessible without this change,

Please note that the older setting `--notebook-dir=$(NOTEBOOKS_PATH_ENV)` was useless as well, since there's no point for us to run jupyter in the `/project/notebooks` working directory